src/cljs/nr/gameboard/card-preview.cljs

```diff
@@ -0,0 +1,35 @@
+(ns nr.gameboard.card-preview
+  (:require [cljs.core.async :refer [put!]]
+            [nr.appstate :refer [app-state]]))
+
+(defn- get-card-data-title [e]
+  (let [target (.. e -target)
+        title (.getAttribute target "data-card-title")]
+    (not-empty title)))
+
+(defn card-preview-mouse-over
+  [e channel]
+  (.preventDefault e)
+  (when-let [title (get-card-data-title e)]
+    (when-let [card (get (:all-cards-and-flips @app-state) title)]
```

Comment:
doesn't matter much but this could be `(get-in @app-state [:all-cards-and-flips title])`